esmvalcore/preprocessor/_multimodel.py

```diff
@@ -223,35 +223,71 @@ def _align(cubes, span):
     return new_cubes
 
 
-def _combine(cubes):
-    """Merge iris cubes into a single big cube with new dimension.
-
-    This assumes that all input cubes have the same shape.
-    """
-    equalise_attributes(cubes)  # in-place
-
-    for i, cube in enumerate(cubes):
-        concat_dim = iris.coords.AuxCoord(i, var_name=CONCAT_DIM)
+def _equalise_cell_methods(cubes):
+    """Equalise coordinates in cubes (in-place)."""
+    # Simply remove all cell methods
+    for cube in cubes:
+        cube.cell_methods = None
```

Comment:
What about ancillary variables? remove_fx_variables handles both.

Reply:
I don't fully get this question. `remove_fx_variables` is called on all cubes in the old and new version of this. Sorry, the diff is a little bit crowded, but the only real change affects `equalise_coordinates`.